source/common/http/conn_manager_utility.cc

```diff
@@ -138,6 +138,7 @@ Network::Address::InstanceConstSharedPtr ConnectionManagerUtility::mutateRequest
     request_headers.removeEnvoyExpectedRequestTimeoutMs();
     request_headers.removeEnvoyForceTrace();
     request_headers.removeEnvoyIpTags();
+    request_headers.removeEnvoyOriginalUrl();
```

Comment:
Can you add a sanitizing test for this in the unit tests for this file? I think there is already a test that covers sanitizing.